Android/build.gradle

```diff
@@ -182,3 +186,7 @@ def getAppKeyPassword() {
 def getAppKey() {
     return hasProperty('ORG_DROIDPLANNER_ANDROID_KEY_ALIAS') ? ORG_DROIDPLANNER_ANDROID_KEY_ALIAS : ''
 }
+
+def getSoloLinkIp(){
+    return hasProperty('SOLO_LINK_IP') ? SOLO_LINK_IP : ''
```

Comment:
@chaviw the build is failing on `travis` because of this line. The empty string should be `'\"\"'` instead.